styles/styles.css

```diff
@@ -1031,6 +1032,19 @@ main .section.float-image-right picture {
   transform: translate(-50%, -50%);
 }
 
+/* For the newsroom featured block title */
+main .newsroom.section .cards-card-body .title {
```

Comment:
`newsroom` seem page specific styles and should likely be moved to a page level css for newroom.

Reply:
Sure Satya. Actually this is for Newsroom Fragment for featured at the top. Now if I put these CSS into newsroom-feed block css, then we may see some kind of Cumulative shift. Let me try that and gauge the behaviour & keep this posted.